pkgs/applications/misc/zathura/djvu/default.nix

```diff
@@ -13,6 +13,13 @@ stdenv.mkDerivation rec {
 
   patches = [ ./gtkflags.patch ];
 
+  patchPhase = stdenv.lib.optionalString stdenv.isDarwin ''
```

Comment:
Is the patchPhase override here intentional? This disables the gtkflags.patch, either way it's better to make that explicit and use `prePatch`/`postPatch` for extra substitutions like this.